dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blobstore.properties

```diff
@@ -1,3 +1,5 @@
+objectstore.implementation=swift
```

Comment:
oh, well, it should be objectstorage and not swift, I missed that previously.
objectstorage is supposed to also handle S3 (see objectorage.provider)